src/utils/components/DiskModal/utils/helpers.ts

```diff
@@ -328,3 +328,17 @@ export const isHotPluggableEnabled = (featureGates: string[]) => {
   }
   return false;
 };
+
+export const convertDataVolumeToTemplate = (
+  dataVolume: V1beta1DataVolume,
+): V1DataVolumeTemplateSpec => ({
+  metadata: dataVolume.metadata,
+  spec: {
+    source: dataVolume.spec.source,
+    sourceRef: dataVolume.spec.sourceRef,
```

Comment:
`metadata` is marked as optional in `V1beta1DataVolume` and `source` and `sourceRef` are marked as optional in `V1beta1DataVolumeSpec`. I know you just moved this pre-existing function, but could you please add the optional chaining operators to those fields?